### PR TITLE
String problems

### PR DESCRIPTION
**Pull request by Prerna6501**

### CODE REVIEW

File: ReplaceSameOccurence [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+int main() {
+    char input1[50], input2[50];
+    printf("Input1(one word) ");
+    scanf("%s", input1);
+    printf("Input2(one word) ");
+    scanf("%s", input2);
+    ToBeReplaced(input1, input2);
+    printf("\nOutput %s", input1);
+    return 0;
```
Comment (Rohit-Vaidya-S) [inline ReplaceSameOccurence:37]: line gap must be added before return 0

File: ATOI [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	int output=0;
+	for (int i = 0; string[i] != '\0'; ++i)
+	{
+		output = output * 10 + string[i] - '0';
+	}
+	printf("using Atoi function;  %d ", output);
+}
+int main()
+{
+	char string[] = "1234567890";
```
Comment (Sanchitha2501) [inline ATOI:13]: Take input from user

File: CSVInput [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#include <stdio.h>
+#include <string.h>
+
+int main() 
+{
+    char inputString[100];
+    printf("Enter string: ");
+    scanf("%[^\n]s",&inputString);
+    char *token;
+    token = strtok(inputString, ",");
```
Comment (Sanchitha2501) [inline CSVInput:11]: try doing it without inbuilt function as shashikala told me the same

File: LexicographicalOrder [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+   for (int i = 0; i <= number; i++)
+   {
+      fgets(str[i], sizeof(str[i]), stdin);
+   }
+   char temp[100];
+   for (int i = 0; i <= number; ++i)
+   {
+      for (int j = i + 1; j <= number; ++j) 
+      {
+         if (strcmp(str[i], str[j]) > 0)
```
Comment (Sanchitha2501) [inline LexicographicalOrder:20]: you have relied much on inbuilt functions,try doing in manually

File: RepeatedCharacter [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    char current = string[i];
+    int count = 0;
+    for (int j = 0; j < length; j++)
+    {
+       if (string[j] == current) 
+       {
+           count++;
+        }
+    }
+    if (count > 1 && strchr(repeat, current) == NULL)
```
Comment (Sanchitha2501) [inline RepeatedCharacter:23]: Try if you can manually do it without using strchr()

File: ATOI [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+	Toint(string);//not using built in 
+	return 0;
```
[the remark this answers is not in the document]
Comment (Prerna6501) [reply ATOI:16]: done

File: CSVInput (hunk 3 above)
Comment (Prerna6501) [reply CSVInput:11]: will do it

File: LexicographicalOrder (hunk 4 above)
Comment (Prerna6501) [reply LexicographicalOrder:20]: strcmp can be used

File: RepeatedCharacter (hunk 5 above)
Comment (Prerna6501) [reply RepeatedCharacter:23]: strchr is a complex function

File: ATOI (hunk 2 above)
Comment (Prerna6501) [reply ATOI]: done

**Pull request closed without merging**